package.json

```diff
@@ -32,5 +32,8 @@
   "homepage": "https://18f.gsa.gov",
   "devDependencies": {
     "pa11y-ci": "^0.3.0"
+  },
+  "dependencies": {
+    "accesslint-cli": "^1.1.1"
```

Comment:
The accesslint rubygem doesn't support using a project specific installation of the node module (yet), so this isn't doing anything. We're considering adding support for this: https://github.com/accesslint/accesslint-ci/issues/46